### DISCUSSION TITLE
I got bus error when I deploy stonedb

### DISCUSSION THREAD

**Discussion by SYuan03** [stoneatom/stonedb#1927] [Q&A]
I follow this instruction https://stonedb.io/zh/docs/getting-started/quick-deployment/deploy-stonedb-with-tar
But I got bus error in step1.5 like this
![image](https://github.com/stoneatom/stonedb/assets/91423966/acf6ddcd-8bc3-4875-bafc-9d9dc00fca83)
I am using the Aliyun server, the configuration is as follows.
![image](https://github.com/stoneatom/stonedb/assets/91423966/da17be02-89a7-40f2-a9e8-4f96860638b6)
I don't know why I got such a bus error. Please help me!

**Comment by Nliver** [comment]
Did you solve the problem?
